README.md

```diff
@@ -1,6 +1,13 @@
 # OpenShift Application Services Connection Operator (RHOAS-Operator)
 
-### Documentation
+OpenShift Application Services Connection Operator (RHOAS-Operator) manages the connections between RHOAS Service Accounts and RHOAS Cloud Services. Users provide RHOAS-Operator with a [cloud.redhat.com](https://cloud.redhat.com) API Token via a secret and the operator uses that token to access the Managed Services APIs on behalf of the user to process their KafkaConnection, CloudServicesRequest, and CloudServiceAccountRequest custom resources. These resources are monitored by the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) which provides connectivity between your application and RHOAS services.
```

Comment:
```suggestion
OpenShift Application Services Connection Operator (RHOAS-Operator) manages the connections between RHOAS Service Accounts and RHOAS Cloud Services. Users provide RHOAS-Operator with a [cloud.redhat.com](https://cloud.redhat.com) API Token via a secret and the operator uses that token to access the Application Services APIs on behalf of the user to process their services like KafkaConnection, CloudServicesRequest, and CloudServiceAccountRequest custom resources. These resources are monitored by the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) which provides connectivity between your application and RHOAS services.
```

Reply:
That is a lot of digest at the start. 

How about we start with something lightweight - 

`RHOAS operator - Connect your Application Services with your OpenShift Cluster`